exir/tests/test_quant_lowering_custom_backend_pass.py

```diff
@@ -506,6 +507,7 @@ class TestQuantLoweringCustomBackendPass(unittest.TestCase):
     def setUp(self) -> None:
         super().setUp()
 
+    @unittest.skipIf(IS_LINUX, "Flaky test on Linux x86, needs debugging")
```

Comment:
Also, if the test is failing consistently (not flaky), a nicer option would be to mark it as XFAIL.  For flaky test, this is the right choice